content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -39,6 +39,12 @@ Users need to be aware of the following when using this feature:
 * CSI drivers may or may not have implemented the volume snapshot functionality. The CSI drivers that have provided support for volume snapshot will likely use the csi-snapshotter. See [CSI Driver documentation](https://kubernetes-csi.github.io/docs/) for details.
 * The CRDs and snapshot controller installations are the responsibility of the Kubernetes distribution.
 
+### Use Cases for This Feature
```

Comment:
I'm thinking even higher up in the intro. Maybe instead of diving head first into apis, the intro should start with some more conceptual information, like:

"The volume snapshot feature allows users to take snapshots of a PersistentVolumeClaim, as well as create a new PersistentVolumeClaim from a snapshot. Some of the use cases this enables include....

To use this feature, the following API resources are used..."

Reply:
So I moved this into the paragraph and nixed the separate section. I can move it all the way to the top and rewrite the first few sentences paragraph if you so desire.

Reply:
Apologies for the delay. Yes, I think it would be great if the use case section could be moved to the very top